src/discussion/serializers.py

```diff
@@ -627,6 +658,11 @@ def get_peer_review(self, obj):
 
         return None
 
+    def get_unified_document(self, obj):
+        from researchhub_document.serializers.researchhub_unified_document_serializer import MinimalUnifiedDocumentSerializer
+        uni_doc = MinimalUnifiedDocumentSerializer(obj.unified_document)
+        return uni_doc.data
+
```

Comment:
What is the purpose of having `unified_document` here? Serializer bloat can become a serious issue when it comes to performance.

If it is necessary, I think using a dynamic serializer here is better than defining a new serializer.